libs/product/src/facades/best-sellers/best-sellers.facade.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { TestBed } from '@angular/core/testing';
+
+import { MockStore } from '@ngrx/store/testing';
+
+import { DaffBestSellersFacade } from './best-sellers.facade';
+import { Store, StoreModule, combineReducers } from '@ngrx/store';
+import { State } from '../../reducers';
+import { cold } from 'jasmine-marbles';
+import { DaffBestSellersLoad, DaffBestSellersLoadSuccess } from '../../actions/best-sellers.actions';
+import { reducers } from '../../reducers';
+
+describe('DaffBestSellersFacade', () => {
+  let store: MockStore<{product: Partial<State>}>;
+  let facade: DaffBestSellersFacade;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports:[
+        StoreModule.forRoot({
+          product: combineReducers(reducers),
+        })
+      ],
+      providers: [
+        DaffBestSellersFacade,
+      ]
+    })
+
+    store = TestBed.get(Store);
+    facade = TestBed.get(DaffBestSellersFacade);
+  });
+
+  it('should be created', () => {
+    expect(facade).toBeTruthy();
+  });
+
+  it('should be able to dispatch an action to the store', () => {
+    spyOn(store, 'dispatch');
+    facade.dispatch({ type: 'SOME_TYPE' });
+    expect(store.dispatch).toHaveBeenCalledTimes(1);
```

Comment:
I think you should do:
```
let action = {type: 'SOME_TYPE'};
facade.dispatch(action);
expect(store.dispatch).toHaveBeenCalledWith(action);
```
just to have a slightly more useful test

Reply:
addressed in 3185d1be860c060dac3f846dca340491ae2aff1a